tests/inference/test_utils.py

```diff
@@ -57,10 +57,10 @@ def test_max_input_len():
 # Add more tests for edge cases
 
 
-def test_get_torch_dtype_cpu_without_ipex(mock_infer_conf):
-    hf_config = None
-    dtype = get_torch_dtype(mock_infer_conf, hf_config)
-    assert dtype == torch.get_default_dtype()
+def test_decide_torch_dtype_cpu_without_ipex(mock_infer_conf):
```

Comment:
can you please cover more test cases in your implementation, for example device is HPU?